test/SILGen/tsan_instrumentation.swift

```diff
@@ -1,9 +1,5 @@
 // RUN: %target-swift-emit-silgen -sanitize=thread %s | %FileCheck %s
-// REQUIRES: tsan_runtime
-
-// FIXME: This should be covered by "tsan_runtime"; older versions of Apple OSs
-// don't support TSan.
-// UNSUPPORTED: remote_run
+// REQUIRES: PTRSIZE=64
```

Comment:
Should this also work on Windows?

Reply:
I think so, yes. If this test wasn't executed on Windows before but is now and fails, then it shows some underlying problem. We never execute/link the test program, but only verify the generated SIL.